test/presenters/return-requirements/frequency-reported.presenter.test.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const FrequencyReportedPresenter = require('../../../app/presenters/return-requirements/frequency-reported.presenter.js')
+
+describe('Frequency Reported presenter', () => {
+  let session
```

Comment:
Same comment as a previous PR. The session can be declared within the next describe block where the context of it is introduced.